### ISSUE TITLE
Voice messages don't load. Need to force shutdown to load them

### ISSUE THREAD

**Issue by dorakas** [signalapp/Signal-Android#8370]
- [x ] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Voice messages don't load. Circle forever. When terminating signal and starting it again. The message gets loaded.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

### Device info

**Device:** pixel 3
**Android version:** 9.0
**Signal version:** 4.27.9

### Link to debug log

https://debuglogs.org/a81201997fd5d4b8801538f973b67f70eed990a65685abd0a333d642a15d10ad

**Comment by jpds** [comment]
Sounds like a dup of #7748.

**Comment by greyson-signal** [comment]
```java
2018-11-15 02:30:44.381 MEZ I Job: [202fb02e-6bd4-4d16-ab65-2699f6181848] AttachmentDownloadJob :: doWork() (Time since submission: 18307694 ms, Run attempt: 55)
2018-11-15 02:30:44.382 MEZ I Job: [202fb02e-6bd4-4d16-ab65-2699f6181848] AttachmentDownloadJob :: Retrying due to unmet requirements. (Time since submission: 18307694 ms, Run attempt: 55)
2018-11-15 02:40:04.449 MEZ I Job: [202fb02e-6bd4-4d16-ab65-2699f6181848] AttachmentDownloadJob :: doWork() (Time since submission: 18867762 ms, Run attempt: 56)
2018-11-15 02:40:04.454 MEZ I Job: [202fb02e-6bd4-4d16-ab65-2699f6181848] AttachmentDownloadJob :: Retrying due to unmet requirements. (Time since submission: 18867763 ms, Run attempt: 56)
```

It looks like what's happening is that you have a screen lock, and the job requires that your phone be unlocked to download the content. However, it keeps trying when your phone _is_ locked, and that increases the retry count and therefore the linear backoff, meaning there's more and more time between retries. By the time you see it, the time between retries could be minutes. 

I'm going to look into this, but to  mitigate it in the meantime, I recommend turning off auto-download for attachments. That way it can't get into this bad state of trying to download something when it can't.

**Comment by greyson-signal** [comment]
Fixed in a3411072ba00ba36465d707d7b1fcaa7910bcd9c.
